contents/handbook/engineering/schema-changes.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Schema changes
+sidebar: Handbook
+showTitle: true
+---
+
+PostHog's database schema evolves constantly along with the app.
+Each schema change safely requires delibration though, as a badly designed migration can cause pain for users,
```

Comment:
```suggestion
Each schema change requires thorough deliberation to make sure it can be safely executed. Badly designed migrations can cause pain for users
```